elfi/examples/gauss.py

```diff
@@ -39,36 +100,73 @@ def ss_var(x):
     return ss
 
 
-def get_model(n_obs=50, true_params=None, seed_obs=None):
-    """Return a complete Gaussian noise model.
+def get_model(true_params=None, cov_ii=None, cov_ij=None, n_obs=15, n_dim=None, seed_obs=None):
+    """Return an initialised Gaussian noise model.
 
     Parameters
     ----------
+    true_params : array_like
+        The first (n - 2) elements corresponds to the mean parameters,
+        and the last two parameters correspond to the variance parameters.
+    cov_ii : None, optional
+        The diagonal variance.
+    cov_ij : None, optional
+        The non-diagonal variance.
     n_obs : int, optional
-        the number of observations
-    true_params : list, optional
-        true_params[0] corresponds to the mean,
-        true_params[1] corresponds to the standard deviation
-    seed_obs : int, optional
-        seed for the observed data generation
+        The number of observations.
+    n_dim : int, optional
+        The number of dimensions.
+    random_state : np.random.RandomState, optional
 
     Returns
     -------
     m : elfi.ElfiModel
 
+    Raises
+    ------
+    ValueError
+        The number of dimension has to match the simulated parameters.
+        A non-trivial check as the variance can be chosen not to be simulated.
+
     """
+    # The default settings use the 2-D Gaussian model:
```

Comment:
I agree that we could switch the default case to the 1-D one. 

Regarding the 1-D Gaussian noise model's separation, it would be obsolete as the 1-D model can be obtained from the general one. We would have more code and files to maintain.